### PR TITLE
feat: add vrslcm product support pack functions

### PR DESCRIPTION
**Pull request by GaryJBlake**
### Summary

- Added `Update-vRSLCMPSPack` to refresh and update Product Support Packs in vRealize Suite Lifecycle Manager.
- Added `Get-vRSLCMPSPack` cmdlet to refresh the metadata for Product Support Packs in vRealize Suite Lifecycle Manager.
- Added `Install-vRSLCMPSPack` cmdlet to install a Product Support Packs in vRealize Suite Lifecycle Manager.

### Type

- [ ] Bugfix
- [x] Enhancement or Feature
- [ ] Code Style or Formatting
- [ ] Documentation
- [ ] Refactoring
- [ ] Chore
- [ ] Other
        Please describe:

### Breaking Changes?

- [ ] Yes, there are breaking changes.
- [x] No, there are no breaking changes.

### Test and Documentation

- [x] Tests have been completed.
- [ ] Documentation has been added or updated.

### Issue References

N/A

### Additional Information

None

### CODE REVIEW

File: PowerValidatedSolutions.psm1 [hunk 1]
```diff
@@ -24802,6 +24854,68 @@ Function Start-vRSLCMProductNode {
...
+    )
+
+    Try {
+        if ($PsBoundParameters.ContainsKey("checkOnline")) {
+            $uri = "https://$vrslcmAppliance/lcm/lcops/api/v2/system-pspack"
+            Invoke-RestMethod $uri -Method 'POST' -Headers $vrslcmHeaders
+        } else {
+            $uri = "https://$vrslcmAppliance/lcm/lcops/api/v2/system-pspack"
+            Invoke-RestMethod $uri -Method 'GET' -Headers $vrslcmHeaders
+        }
```
Comment (tenthirtyam) [inline PowerValidatedSolutions.psm1:24885]: Do you want to add a `} Catch {` here?

File: PowerValidatedSolutions.psm1 [hunk 2]
```diff
@@ -24802,6 +24854,68 @@ Function Start-vRSLCMProductNode {
...
+    Param (
+        [Parameter (Mandatory = $false)] [ValidateNotNullOrEmpty()] [String]$pspackId
+    )
+
+    Try {
+        
+        $uri = "https://$vrslcmAppliance/lcm/lcops/api/v2/system-pspack/$pspackId"
+        Invoke-RestMethod $uri -Method 'POST' -Headers $vrslcmHeaders
+    }
+    Catch {
```
Comment (tenthirtyam) [inline PowerValidatedSolutions.psm1:24914]: `} Catch {`

File: PowerValidatedSolutions.psm1 [hunk 3]
```diff
@@ -24802,6 +24854,68 @@ Function Start-vRSLCMProductNode {
 }
 Export-ModuleMember -Function Start-vRSLCMProductNode
 
+Function Get-vRSLCMPSPack {
```
Comment (tenthirtyam) [inline PowerValidatedSolutions.psm1:24857]: Do you want to update function details to "Product Support Packs" in these two functions to align with the user functions?

**Review by tenthirtyam — APPROVED** [review]
LGTM!

File: PowerValidatedSolutions.psm1 (hunk 1 above)
Comment (GaryJBlake) [reply PowerValidatedSolutions.psm1:24885]: We have a catch already

File: PowerValidatedSolutions.psm1 (hunk 3 above)
Comment (GaryJBlake) [reply PowerValidatedSolutions.psm1:24857]: Good call

**Pull request merged**